x/swaprouter/creator_test.go

```diff
@@ -42,8 +45,13 @@ func (suite *KeeperTestSuite) TestCreatePool() {
 			msg:                validBalancerPoolMsg,
 			expectedModuleType: gammKeeperType,
 		},
+		{
```

Comment:
I'm worried about this getting missed. Can you please add any swaprouter changes while the merge is in progress to this issue: https://github.com/osmosis-labs/osmosis/issues/3556. PR link with a change summary would be ideal